### DISCUSSION TITLE
使用自己的nginx后，如何去掉 caddy 相关的部分

### DISCUSSION THREAD

**Discussion by JinPengGeng** [Mereithhh/vanblog#53] [Q&A]
我的服务器中，会有多个项目同时部署，需要统一的入口，已经开启了 https，用不到 caddy

1. caddy 是深度依赖的吗？
2. 如何去掉 caddy，使用自己的代理
3. 热更新能排在最前面吗，现在想用，但更新太频繁，不太敢尝试

**Comment by Mereithhh** [comment]
## 问题一
具体请看这个 Issue #51 
## 问题二
您完全不用考虑 Caddy，你把整个 VanBlog 当成一个整体，对外映射一个非常用的端口，正常参考文档中的反代配置进行反代即可。
## 问题三
热更新的话我会尽量往前做，但目前可以用一键部署脚本了。 您可以用一键部署脚本部署，设置非常用端口，然后正常用您熟悉的反代进行反代即可。更新的话一键脚本可以一键更新，很方便。

![image](https://user-images.githubusercontent.com/22872368/191295959-baa66554-64ab-4956-96bd-054dd0af2e8d.png)

**Reply by Mereithhh** [reply]
我在文档中补充了关于 caddy 的问题，以免有一些人误解反代的复杂性。
